config/pretty-routes.php

```diff
@@ -88,4 +114,41 @@
      */
 
     'locale_force' => false,
+
+    /*
+     * Enable open link in new tab.
+     *
+     * Useful for manual testing or visual checks.
+     *
+     * This option related with "dummy_variable_prefix"
+     */
+
+    'show_path_link' => true,
+
+    /*
+     * Laravel stores variable names with curly bracktes, this option replaces "prefix" and "variable name".
+     * So we can prevent url encode when browsing and we can get readable urls
+     *
+     * For example
+     * before admin/line/lines/{line_id} => admin/line/lines/%7Bline_id%7D
+     * after: admin/line/lines/{line_id} => admin/line/lines/param_line_id
+     *
+     * This option depends to "show_path_link"
+     */
+
+    'dummy_variable_prefix' => 'param_',
+
+    /*
+     * Click and copy to clipboard "path" and "name" text.
+     */
+
+    'click_and_copy' => true,
+
+    /*
+     * Double click and copy to clipboard "path" and "name" text.
+     *
+     * This option overwrites "click_and_copy" option.
+     */
+
+    'double_click_and_copy' => true,
```

Comment:
Tell me what should happen on double click?
I check the work, and when you double-click the script reads them as two single clicks, remembering the path or name in the clipboard.

Reply:
@EgoistDeveloper, ping :)